src/kili/services/copy_project/__init__.py

```diff
@@ -173,17 +175,15 @@ def _copy_assets(self, from_project_id: str, new_project_id: str) -> None:
             "jsonMetadata",
         )
 
-        def download_and_upload_assets(assets: List[Dict]) -> List[Dict]:
-            with TemporaryDirectory() as tmp_dir:
-                downloaded_assets = self._download_assets(from_project_id, fields, tmp_dir, assets)
-                return self._upload_assets(new_project_id, downloaded_assets)
+        assets_gen = self.kili.kili_api_gateway.list_assets(filters, fields, options)
 
-        asset_gen = self.kili.kili_api_gateway.list_assets(
-            filters, fields, options, download_and_upload_assets
-        )
-        # Generator needs to be iterated over to actually fetch assets
-        for _ in asset_gen:
-            pass
```

Comment:
make sure that copy project e2e tests pass, I remember we had issues with this

Reply:
j'ai lancé les tests E2E manuellement

Reply:
https://github.com/kili-technology/kili-python-sdk/actions/runs/6110561997